### PR TITLE
OPS: Replace deprecated Clipboard

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/9mkffmunyqc3qpvv61n0tcm4n0

[download apk](http://filestorage.bluewallet.io:1488/1591737465.apk)

File: screen/send/psbtWithHardwareWallet.js [hunk 1]
```diff
@@ -8,14 +8,14 @@ import {
...
   TextInput,
-  Clipboard,
   Linking,
   Platform,
+  Text,
   PermissionsAndroid,
   StyleSheet,
 } from 'react-native';
 import QRCode from 'react-native-qrcode-svg';
-import { Text } from 'react-native-elements';
```
Comment (Overtorment) [inline screen/send/psbtWithHardwareWallet.js:18]: you removed import of `Text` and its being used below. will it crash?

Comment (marcosrdz) [reply screen/send/psbtWithHardwareWallet.js:18]: It was using element’s text instead of RN’s text.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dvry737nb64zx8xp1bpha2k5c4

[download apk](http://filestorage.bluewallet.io:1488/1591909660.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5hde9kbkm3nnmh93zd4ayyc2e4

[download apk](http://filestorage.bluewallet.io:1488/1591911127.apk)

File: screen/send/psbtWithHardwareWallet.js (hunk 1 above)
Comment (Overtorment) [reply screen/send/psbtWithHardwareWallet.js:18]: But I don’t see Text import from react-native..?

**Pull request merged**